client/reader/following-edit/style.scss

```diff
@@ -146,7 +146,7 @@
 
 	.card.is-search-result {
 		position: absolute;
-			top: 51px;
+		top: 51px;
```

Comment:
[Reference](https://github.com/Automattic/wp-calypso/blob/bdf36c9d45793157282b402f277bda6d471d2694/docs/coding-guidelines/css.md#positioning).  The idea is something like "these attributes are nested under the position" which makes conceptual sense to me.